src/libs/actions/Search.ts

```diff
@@ -83,4 +87,27 @@ function deleteMoneyRequestOnSearch(hash: number, transactionIDList: string[]) {
     API.write(WRITE_COMMANDS.DELETE_MONEY_REQUEST_ON_SEARCH, {hash, transactionIDList}, {optimisticData, finallyData});
 }
 
-export {search, createTransactionThread, deleteMoneyRequestOnSearch, holdMoneyRequestOnSearch, unholdMoneyRequestOnSearch};
+type Params = Record<string, string | string[]>;
+
+function exportSearchItemsToCSV(query: string, reportIDList: Array<string | undefined> | undefined, transactionIDList: string[], policyIDs: string[], onDownloadFailed: () => void) {
+    const fileName = `Expensify_${query}.csv`;
```

Comment:
Hmm I thought we were gonna use the export date here?

Reply:
The `fileDownload` function automatically adds the date to the file name by calling [appendTimeToFileName](https://github.com/Expensify/App/blob/78ac528d1c0699aa5cf58652973d3ac52e993b33/src/libs/fileDownload/FileUtils.ts#L154).

Reply:
Cool, why are we appending the query to the name then? We'll have to change this soon once we introduce search syntax

Reply:
It is just what we agreed on the doc, but I don't mind changing it to `Expensify.csv`

Reply:
I think we should change it since it'll have to be done sooner or later